app/ui/legacy/src/main/java/com/fsck/k9/ui/messageview/MessageTopView.kt

```diff
@@ -257,7 +258,7 @@ class MessageTopView(
             return false
         }
         val senderEmailAddress = getSenderEmailAddress(message) ?: return false
-        val contacts = Contacts.getInstance(context)
+        val contacts = DI.get(Contacts::class.java)
```

Comment:
Please inject via property using `by inject()`

Reply:
Then the MessageTopView needs to be changed to a KoinComponent, too.

Reply:
Done